sdk/python/feast/errors.py

```diff
@@ -22,6 +22,16 @@ def __init__(self, name, project=None):
             super().__init__(f"Entity {name} does not exist")
 
 
+class FeatureServiceNotFoundException(FeastObjectNotFoundException):
+    def __init__(self, name, project=None):
+        if project:
+            super().__init__(
+                f"FeatureService {name} does not exist in project {project}"
```

Comment:
Can we use `Feature service` instead of the class name? I'm doing that (and `feature view` `feature table`) throughout docs and code.